tests/test_tem.py

```diff
@@ -1,11 +1,11 @@
 """Unit test for TEM Simulator wrapper."""
-from simSPI import tem
-
-import tempfile
 import os
+import tempfile
 
 import yaml
 
+from simSPI import tem
+
 
 def test_tem_main():
```

Comment:
are these tests running tem simulator on github? how since I thought there needed to be a local install? 

if the tests are in a try catch, I think that's a good fix for now, but you could just put a note in the documentation about that. something like "sim.run in try catch because requires local install and currently no containerization is set up")

Reply:
I agree, it would help to know what is being tested.

Reply:
These changes seem to be from one of the merges. This test_tem.py is essentially the same as the one in the main branch right now. It was put in place to provide code coverage for the _function definitions_ not the implementation of the tem_wrapper, hence it passes. The actual tem_implementation with associated tests will be merged with #32.

Reply:
ok let's merge for now. but I stand by my point